src/sentry/api/endpoints/org_auth_tokens.py

```diff
@@ -71,6 +72,18 @@ def post(self, request: Request, organization: Organization) -> Response:
             data=token.get_audit_log_data(),
         )
 
+        # Notify all owners of org of new token
+        owners = organization.get_owners()
```

Comment:
This won't work, as you won't have access to django' Organization in a control silo endpoint. Instead you need to use the `OrganizationMemberMapping` table to get membership data, which you can then use to read the related users.

```python
from sentry.models.organizationmembermapping import OrganizationMemberMapping
from sentry.models.user import User

members = OrganizationMemberMapping.filter(organization_id=organization.id, role="owner")
owners = User.objects.filter(user_id=[m.user_id for m in members])
```

Should work.